lib/vmpooler/pool_manager.rb

```diff
@@ -867,16 +868,12 @@ def sleep_with_wakeup_events(loop_delay, wakeup_period = 5, options = {})
 
             if options[:clone_target_change]
               clone_target = redis.hget('vmpooler__config__clone_target}', options[:poolname])
-              if clone_target
-                break unless clone_target == initial_clone_target
-              end
+              break if clone_target && !clone_target == initial_clone_target
```

Comment:
unless clone_target is a Boolean, this is not the same. ! on a string returns false.
```suggestion
              break if clone_target && clone_target != initial_clone_target
```